widget/embedded/src/store/ui.ts

```diff
@@ -8,14 +12,45 @@ interface UiState {
   currentPage: string;
   setSelectedSwap: (requestId: string | null) => void;
   setCurrentPage: (path: string) => void;
+  isActiveTab: boolean | null;
```

Comment:
Let's make it clearer by only keeping a boolean value to check if the tab is active or not and keeping null which is using for checking initializing phase separate.
The condition will be like:
```
state != 'not-init' && !isActive
```